third_party/brotli/c/common/platform.h

```diff
@@ -293,21 +303,21 @@ To apply compiler hint, enclose the branching condition into macros, like this:
 /* Portable unaligned memory access: read / write values via memcpy. */
 static BROTLI_INLINE uint16_t BrotliUnalignedRead16(const void* p) {
   uint16_t t;
-  memcpy(&t, p, sizeof t);
+  MEMCPY_PLATFORM(&t, p, sizeof t);
   return t;
 }
 static BROTLI_INLINE uint32_t BrotliUnalignedRead32(const void* p) {
   uint32_t t;
-  memcpy(&t, p, sizeof t);
+  MEMCPY_PLATFORM(&t, p, sizeof t);
   return t;
 }
 static BROTLI_INLINE uint64_t BrotliUnalignedRead64(const void* p) {
   uint64_t t;
-  memcpy(&t, p, sizeof t);
+  MEMCPY_PLATFORM(&t, p, sizeof t);
   return t;
 }
 static BROTLI_INLINE void BrotliUnalignedWrite64(void* p, uint64_t v) {
-  memcpy(p, &v, sizeof v);
+  MEMCPY_PLATFORM(&t, p, sizeof t);
```

Comment:
Should the actual parameters here change from (p, &v, sizeof v) to (&t, p, sizeof t)?

Reply:
Oops, should be (p, &v, sizeof v). Great catch!